apps/pdc-frontend/src/app/[locale]/[...not-found]/page.tsx

```diff
@@ -26,7 +26,9 @@ const NotFoundPage = async ({ params: { locale } }: { params: { locale: string }
         ]}
       />
       <PageTitle>{data?.notFoundPage?.data?.attributes?.title}</PageTitle>
-      <Markdown>{data?.notFoundPage?.data?.attributes?.body}</Markdown>
+      <Markdown imageUrl={getImageBaseUrl()} locale={locale}>
```

Comment:
I would maybe consider keeping a wrapper Markdown component in the PDC package so you don't have to define the `imageUrl` everywhere it's used. Something like `<PdcMarkdown>` or `<MarkdownWithImage>` that internally sets `imageUrl` to however the project wants to build its url.

Reply:
Maybe this also applies to `locale` for PDC

Reply:
I understand your point. However, when I attempted to implement this, I encountered some issues. The `STRAPI_PUBLIC_URL` environment variable only functions on the server side, while the Markdown component only works on the client side. I found two potential solutions:
1. Continue with the current approach.
2. Create a new environment variable that functions on the client side, beginning with `NEXT_PUBLIC_`.
Create a new env variable it's not really great because we have to pass it to the DevOps teams 
https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser

Reply:
The `locale` property is only for the PDC. That's why I created a custom markdown component on the PDC side.https://github.com/frameless/strapi/blob/f2876e90ba9bd5b609660f6b508a86accee49183/packages/ui/src/components/Markdown/index.tsx